lib/hardware_services/inventory_drivers/hil.yml

```diff
@@ -0,0 +1,2 @@
+url: http://127.0.0.1:5000    # the address of HIL server
```

Comment:
@vsemp see #54 
as we discussed prior, I added this as a parameter to conf/quads.yml